app/assets/javascripts/school_administrator_dashboard/dashboard_components/discipline_dashboard/SchoolDisciplineDashboard.js

```diff
@@ -143,7 +139,7 @@ class SchoolDisciplineDashboard extends React.Component {
     const categories = this.sortChartKeys(Object.keys(selectedChart.disciplineIncidents));
     const seriesData = categories.map((type) => {
       if (!selectedChart.disciplineIncidents[type]) return [];
-      const incidents = this.filterIncidentDates(selectedChart.disciplineIncidents[type]);
+      const incidents = selectedChart.disciplineIncidents[type];
```

Comment:
Ah so this is the import change, right?  I'm still a little confused because the PR descriptions are about improving sorting, but these changes all seem related to filtering.  Is the change that it was always sorting before, just not on the right subset of events based on the current filtering?